src/main.tsx

```
@@ -1,4 +1,255 @@
+  }
+
+  return (
+    <div id="comment-container">
+      <form onSubmit={onSubmit}>
+        <label>Comment</label>
+        <input type="text" name="content" />
+        <button type="submit">Submit</button>
+      </form>
+      <div id="comment-list">
+        {commentList &&
+          commentList.map((comment) => <span>{comment.content}</span>)}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)